.secrets

```diff
@@ -0,0 +1,2 @@
+NETLIFY_AUTH_TOKEN=M58Tzg4POsefYEB0uzxRM9L6ydsW3jh80Np9Ogdc7aY
```

Comment:
:grimacing: mistake? Or are these your personal tokens? (if so I suggest renaming the file to something like `.example_secrets`)